packages/core/src/types.ts

```diff
@@ -42,17 +41,27 @@ export type AnonAadhaarPCDProof = {
 }
 
 /**
- * @dev witness use for create zk proof of AnonAadhaarPCD package.
+ * @dev Arguments needed to compute the witness.
  */
-export type AnonAadhaarPCDArgs = {
+export type AnonAadhaarArgs = {
   aadhaarData: StringArrayArgument // private
   aadhaarDataLength: NumberArgument // private
   signature: StringArrayArgument // private
   pubKey: StringArrayArgument // public
   signalHash: StringArgument // public
 }
 
-export type Proof = [
+/**
+ * @dev Arguments that needs to be extracted to generates the arguments of the witness.
+ */
+export type WitnessQRInputs = {
+  paddedMessage: string[]
```

Comment:
WE should change WitnessQRInputs have same properties with AnonAadhaarArgs

Reply:
I will remove it, as it is no longer used.